irods/client_configuration/__init__.py

```diff
@@ -61,6 +68,19 @@ def _var_items(root):
     return []
 
 def save(root = None, string='', file = ''):
+    """Save the current configuration.
+
+    When called simply as save(), simply writes all client settings to a configuration file.
+
+    The 'root' and 'string' parameters are not likely to be overridden when called from
+    applications.  They vary only in save()'s internal recursion.
+    'root' specifies at which subtree node to start writing.  None denotes the top level.
+    'string' specifies the dotted prefix and should be empty for the top level.
+
+    The configuration file path will normally be the string value of DEFAULT_CONFIG_PATH,
+    but this can be overridden by supplying a nonzero string in the 'file' parameter.
+
```

Comment:
Remove?

Reply:
Wait - not clear what needs removing? The Whole doc string?

Reply:
I'm referring to the empty line on line 82.

Reply:
Ah ok .. sorry. Phone gui a bit unclear :/

Reply:
All good.

Reply:
done